exampleService/native/src/impl/ExampleService.cpp

```diff
@@ -111,33 +113,39 @@ namespace example {
          }
       }
 
-      float ExampleService::getCpuNextValue(std::vector<float>& vec) {
+      float ExampleService::GetCpuNextValue(std::vector<float>& vec) {
          static std::size_t index_cpu = 0;
          if (index_cpu >= vec.size()) {
             index_cpu = 0;
          }
          return vec[index_cpu++];
       }
 
-      float ExampleService::getGpuNextValue(std::vector<float>& vec) {
+      float ExampleService::GetGpuNextValue(std::vector<float>& vec) {
          static std::size_t index_gpu = 0;
          if (index_gpu >= vec.size()) {
             index_gpu = 0;
          }
          return vec[index_gpu++];
       }
 
-      float ExampleService::getAmbientNextValue(std::vector<float>& vec) {
+      float ExampleService::GetAmbientNextValue(std::vector<float>& vec) {
          static std::size_t index_ambient = 0;
          if (index_ambient >= vec.size()) {
             index_ambient = 0;
          }
          return vec[index_ambient++];
       }
 
-      float ExampleService::calculateMaxValue(const std::vector<float>& vec) {
+      float ExampleService::CalculateMaxValue(const std::vector<float>& vec) {
          float max = vec[0]; // Initialize max to the first element of the vector
          // Iterate through the vector to find the maximum element
+         // for (const auto& i.index = 1 : vec) {
+         //    if (i > max) {
+         //          max = i;
+         //    }
+         // }
+
          for (size_t i = 1; i < vec.size(); ++i) {
```

Comment:
Pentru determinarea elementului maxim dintr-un std container putem folosi functii deja implementate in libraria standard:
https://en.cppreference.com/w/cpp/algorithm/max_element
Folosirea acestor functii creste "code readability"-ul si reduce sansa de a introduce bug-uri atunci cand re-implementam o functionalitate existenta.